test/functional/test_framework/test_node.py

```diff
@@ -155,13 +163,16 @@ def dispatcher(*args, **kwargs):
 
     def send_cli(self, command, *args, **kwargs):
         """Run bitcoin-cli command. Deserializes returned string as python object."""
-
+        input = None
+        if "input" in kwargs:
+            input = kwargs["input"]
+            del kwargs["input"]
```

Comment:
In commit "[test] Add support for custom arguments to TestNodeCLI"

You could replace these four lines with `input = kwargs.pop("input", None)`, or change the function signature to `def send_cli(self, command, *args, input=None, **kwargs)` to get the same result.

Probably though it is better just to pass input to the TestNodeCLI constructor as John suggests.